model/src/main/kotlin/licenses/LicenseInfoResolver.kt

```diff
@@ -207,7 +207,7 @@ class LicenseInfoResolver(
         }.forEach { provenance ->
             val archiveDir = createTempDirectory("$ORT_NAME-archive").toFile().apply { deleteOnExit() }
 
-            if (!archiver.unarchive(archiveDir, id, provenance)) return@forEach
+            if (!archiver.unarchive(archiveDir, provenance)) return@forEach
```

Comment:
Commit message: s/idependent/independent/

> Make the file archive paths independent of the identifier to avoid storing duplicate archives

Just curious about this rationale. Is it really likely that omitting the identifier avoids any duplicates? IIUC this would only happen if the provenance is the same, but the identifier would be different. And that in turn could only happen for repos that host multiple packages. But for these the VCS info should also contain a path, which means provenance is already different.

Don't get me wrong, I agree that just using the provenance here is sufficient. I'm just wondering whether the rationale is correct.

Reply:
The file archiver does not adhere to VCS path anymore intentionally. Thus, it avoids duplicates if multiple packages are release from the same repository and revision, if if the vcs paths of the packages are set differently.

Reply:
> Commit message: s/idependent/independent/

That typo is still there.

> The file archiver does not adhere to VCS path anymore intentionally.

Can you help me remember how this works now? Does the archiver always archive all license-ish files, but a later logic (which one?) only takes those license files into account that are relevant for a specific package's directory in the repository?

Reply:
Yes, it's like you just wrote. So

1. the spare checkout checks out all license-ish file no matter of the VCS path using configured license file patterns
2. the file archiver archives using the patterns, no matter of the VCS path
3. The applicable license files are determined by the `RootLicenseMatcher` class. Also see its call hierarchy for the components using it.

Reply:
The commit message is now fixed.

Reply:
@sschuberth just FYI as a follow up to above point regarding duplication: We went from 130953 to 29126 archives.